src/main/java/be/acara/frontend/controller/OrderController.java

```diff
@@ -49,4 +54,11 @@ public String displayOrderForm(@PathVariable("eventId") Long eventId, Model mode
         );
         return "buyOrder";
     }
+
+    @GetMapping(value = "/ticket/{eventId}", produces = MediaType.APPLICATION_PDF_VALUE)
+    @ResponseBody
+    public HttpEntity<byte[]> downloadEventTicket(@PathVariable("eventId") Long eventId, HttpServletResponse response) {
+        response.setHeader("Content-Disposition", "attachment; filename=acara_events_ticket.pdf");
+        return new HttpEntity(orderService.getEventTicket(eventId).getTicket());
+    }
```

Comment:
[Raw use of parameterized class 'HttpEntity'](https://i.imgur.com/NbPNCqO.png)

Possibly replace with this?
```
return ResponseEntity.ok()
                .header("Content-Disposition", "attachment; filename=acara_events_ticket.pdf")
                .body(orderService.getEventTicket(eventId).getTicket());
```